packages/kbn-test/src/functional_tests/lib/run_elasticsearch.ts

```diff
@@ -7,20 +7,32 @@
  */
 
 import { resolve } from 'path';
+import type { ToolingLog } from '@kbn/dev-utils';
 import { KIBANA_ROOT } from './paths';
-import { createLegacyEsTestCluster } from '../../legacy_es';
+import type { Config } from '../../functional_test_runner/';
+import { createTestEsCluster } from '../../es';
 
 import { setupUsers, DEFAULT_SUPERUSER_PASS } from './auth';
 
-export async function runElasticsearch({ config, options }) {
+interface RunElasticsearchOptions {
+  log: ToolingLog;
+  esFrom: string;
+}
+export async function runElasticsearch({
+  config,
+  options,
+}: {
+  config: Config;
+  options: RunElasticsearchOptions;
+}) {
   const { log, esFrom } = options;
   const ssl = config.get('esTestCluster.ssl');
```

Comment:
I hope we migrate to `@kbn/config-schema` one day not to lose type safety here https://github.com/elastic/kibana/issues/69393